bikeshed/include/html/header.include

```diff
@@ -2,13 +2,11 @@
 <html lang="en">
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  <title>[TITLE] [LEVEL]</title>
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="w3c-status" content="[STATUS]">
-  <link href="../default.css" rel=stylesheet type="text/css">
-  <link href="../csslogo.ico" rel="shortcut icon" type="image/x-icon">
-  <link href="styles/styles-html.css" rel=stylesheet type="text/css">
-  <link href="[W3C-STYLESHEET-URL]" rel=stylesheet type="text/css">
+  <title>[TITLE] [LEVEL]</title>
```

Comment:
The problem is the spec name is HTML the Level is 5.1. We want to define those as separate entities for use in other places in the document. However the `<title>` and the `<h1>` in the include file need to use the combined of what `<h1>`, and `Level:` defines in the `.bs` file. I would prefer to leave this alone for now since I am not doing major edits to the spec.